docs/Developers/Language_Naming_Format.rst

```diff
@@ -0,0 +1,9 @@
+.. important::
+
+    - `For language naming always consider the format that is used in the`
+      `corresponding Wikipedia page, (exception: use CPP instead`
+      `of cpp or c++);`
+    - `If the definition does not exist already a language definition`
+      `should be added while writing new bear;`
+    - `The alias used in LANGUAGES should be one of the`
+      `names in the coala definition.`
```

Comment:
Can you please explain why the lines have been surrounded by backticks?

Reply:
@ayan-b I thought they are necessary for the bullet points but I came to know that they are not :sweat_smile: I have fixed it